bot/cogs/random_cog/random_cog.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import random
+import time
+
+import discord
+import discord.ext.commands as commands
+
+from bot.consts import Colors
+
+log = logging.getLogger(__name__)
+
+
+class RandomCog(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def flip(self, ctx):
+
+        random.seed(time.time())
+
+        embed = discord.Embed(title="Coin Flip", color=Colors.ClemsonOrange)
+
+        heads = discord.File(filename="Heads.jpg",
+                             fp="bot/cogs/random_cog/assets/Heads.jpg")
```

Comment:
only thing is the string " " instead of ' ' , other then that looks good

Reply:
Fixed it